scalafmt-tests/shared/src/test/resources/scala3/FewerBraces.stat

```diff
@@ -2317,3 +2317,435 @@ object a:
+indent.fewerBraces = never
+===
+@main def main() =
+  val first =
+    for i <- 0 to 10
+    yield i
+  .map(_ * 2)
+  .last
+  println(first)
+>>>
+@main def main() =
+   val first =
+      for i <- 0 to 10
+      yield i
+   .map(_ * 2)
+     .last
+   println(first)
+<<< #4614 for-yield then select, always
+indent.fewerBraces = always
+===
+@main def main() =
+  val first =
+    for i <- 0 to 10
+    yield i
+  .head
+  println(first)
+>>>
+@main def main() =
+   val first =
+        for i <- 0 to 10
+        yield i
+     .head
+   println(first)
+<<< #4614 for-yield then apply, always
+indent.fewerBraces = always
+===
+@main def main() =
+  val first =
+    for i <- 0 to 10
+    yield i
+  .map(_ * 2)
+  println(first)
+>>>
+@main def main() =
+   val first =
+        for i <- 0 to 10
+        yield i
+     .map(_ * 2)
+   println(first)
+<<< #4614 for-yield then apply and select, always
+indent.fewerBraces = always
+===
+@main def main() =
+  val first =
+    for i <- 0 to 10
+    yield i
+  .map(_ * 2)
+  .last
+  println(first)
+>>>
+@main def main() =
+   val first =
+        for i <- 0 to 10
+        yield i
+     .map(_ * 2)
+     .last
+   println(first)
+<<< #4614 for-yield then select, beforeSelect
+indent.fewerBraces = beforeSelect
+===
+@main def main() =
+  val first =
+    for i <- 0 to 10
+    yield i
+  .head
+  println(first)
+>>>
+@main def main() =
+   val first =
+        for i <- 0 to 10
+        yield i
+     .head
+   println(first)
+<<< #4614 for-yield then apply, beforeSelect
+indent.fewerBraces = beforeSelect
+===
+@main def main() =
+  val first =
+    for i <- 0 to 10
+    yield i
+  .map(_ * 2)
+  println(first)
+>>>
+@main def main() =
+   val first =
+        for i <- 0 to 10
+        yield i
+     .map(_ * 2)
+   println(first)
+<<< #4614 for-yield then apply and select, beforeSelect
+indent.fewerBraces = beforeSelect
+===
+@main def main() =
+  val first =
+    for i <- 0 to 10
+    yield i
+  .map(_ * 2)
+  .last
+  println(first)
+>>>
+@main def main() =
+   val first =
+        for i <- 0 to 10
+        yield i
+     .map(_ * 2)
+     .last
+   println(first)
+<<< #4614 infix then select, never
+indent.fewerBraces = never
+===
+@main def main() =
+  val first =
+    foo :: bar ::
+    baz
+  .head
+  println(first)
+>>>
+@main def main() =
+   val first =
+      foo :: bar ::
+        baz
+   .head
+   println(first)
+<<< #4614 infix then apply, never
+indent.fewerBraces = never
+===
+@main def main() =
+  val first =
+    foo :: bar ::
+    baz
+  .map(_ * 2)
+  println(first)
+>>>
+@main def main() =
+   val first =
+      foo :: bar ::
+        baz
+   .map(_ * 2)
+   println(first)
+<<< #4614 infix then apply and select, never
+indent.fewerBraces = never
+===
+@main def main() =
+  val first =
+    foo :: bar ::
+    baz
+  .map(_ * 2)
+  .last
+  println(first)
+>>>
+@main def main() =
+   val first =
+      foo :: bar ::
+        baz
+   .map(_ * 2)
+     .last
```

Comment:
This is not a perfect indent, but I think this is such a rare case that we might not want to bother with it